src/assets/sass/protocol/includes/_themes.scss

```diff
@@ -6,6 +6,34 @@
 // themes. Mixins and functions can draw from these variable maps, swapping to
 // a different map based on the value of the global $brand-theme variable.
 
+//* -------------------------------------------------------------------------- */
+// Vertical and Horizontal Spacing
+
+
+$theme-spacing-firefox: (
+    // xs
+    v-grid-xs: $layout-md,
+    h-grid-xs: $layout-xs,
+
+    // sm is the same as xs
+    v-grid-sm: $layout-md,
+    h-grid-sm: $layout-xs,
+
+    // md
+    v-grid-md: $layout-md,
+    h-grid-md: $layout-md,
```

Comment:
Just noticed: Should this be `$layout-lg` (64px)? Referring to https://github.com/mozilla/protocol/issues/345#issuecomment-584856741

Reply:
Oh, yes.